spec/spec_helper.rb

```diff
@@ -1,8 +1,9 @@
-#This file is generated by ModuleSync, do not edit.
+# This file is generated by ModuleSync, do not edit.
 require 'puppetlabs_spec_helper/module_spec_helper'
 
 # put local configuration and setup into spec_helper_local
 begin
   require 'spec_helper_local'
 rescue LoadError
+  puts 'Spec Helper LoadError'
```

Comment:
I commented on this in the motd PR https://github.com/puppetlabs/puppetlabs-motd/pull/85

Reply:
Thanks Eric, have used this here. Great suggestion!